options.go

```diff
@@ -268,3 +272,10 @@ func WithEdgeTriggeredIO(et bool) Option {
 		opts.EdgeTriggeredIO = et
 	}
 }
+
+// WithEdgeTriggeredIOSpace set number of bytes that gnet can read/write up to in one event loop of ET.
```

Comment:
// WithEdgeTriggeredIOSpace sets the number of bytes that gnet can read/write up to in one event loop of ET.